types.go

```diff
@@ -673,7 +673,9 @@ func readPLPType(ti *typeInfo, r *tdsBuffer) interface{} {
 		// size unknown
 		buf = bytes.NewBuffer(make([]byte, 0, 1000))
 	default:
-		buf = bytes.NewBuffer(make([]byte, 0, size))
+		// Uses a fixed buffer size to avoid overflow.
```

Comment:
I would add more details why this is needed and why we don't want to trust the `size := r.uint64()` value.